playwright/element-web-test.ts

```diff
@@ -155,11 +158,15 @@ export const test = base.extend<
 
                 // Ensure the language is set to a consistent value
                 window.localStorage.setItem("mx_local_settings", '{"language":"en"}');
+
+                // Enable the lab features
+                for (const feature of labsFlags) {
+                    window.localStorage.setItem(`mx_labs_feature_${feature}`, "true");
+                }
```

Comment:
OOI what's the rationale for doing it like this vs via config like we do for rust crypto etc

Reply:
no particular reason, the previous method was setting localstorage so I ended up with this.
I've changed this to override config.json in the page fixture.